docs/getting_started.md

```diff
@@ -82,68 +46,71 @@ $ sudo mv ./clusterctl.darwin_amd64 /usr/local/bin/clusterctl
+* **provider-components.yaml**: a YAML manifest which adds all the Cluster API and CAPV-specific resources to your management cluster
+* **addons.yaml**: a YAML manifest indicating any additional addons you want on your management cluster (e.g. CNI plugin)
+
+The project Makefile provides a convenient make target to generate the manifests. Before attempting to generate the above manifests, the following environment variables should be set based on your vSphere environment:
 
-```bash
+```shell
+$ cat <<EOF >envvars.txt
 # vCenter config/credentials
-export VSPHERE_SERVER=10.0.0.1                # (required) The vCenter server IP or UR
-export VSPHERE_USER=viadmin@vmware.local      # (required) The vCenter user to login with
-export VSPHERE_PASSWORD=some-secure-password  # (required) The vCenter password to login with
+export VSPHERE_SERVER='10.0.0.1'                # (required) The vCenter server IP or UR
+export VSPHERE_USER='viadmin@vmware.local'      # (required) The vCenter user to login with
+export VSPHERE_PASSWORD='some-secure-password'  # (required) The vCenter password to login with
 
 # vSphere deployment configs
-export VSPHERE_DATACENTER="SDDC-Datacenter"         # (required) The vSphere datacenter to deploy the management cluster on
-export VSPHERE_DATASTORE="DefaultDatastore"         # (required) The vSphere datastore to deploy the management cluster on
-export VSPHERE_NETWORK="vm-network-1"               # (required) The VM network to deploy the management cluster on
-export VSPHERE_RESOURCE_POOL="*/CAPV"               # (required) The vSphere resource pool for your VMs
-export VSPHERE_FOLDER="Workloads"                   # (optional) The VM folder for your VMs, defaults to the root vSphere folder if not set.
-export VSPHERE_TEMPLATE="ubuntu-1804-kube-v1.13.6"  # (required) The VM template to use for your management cluster.
-export VSPHERE_DISK_GIB="50"                        # (optional) The VM Disk size in GB, defaults to 20 if not set
-export VSPHERE_NUM_CPUS="2"                         # (optional) The # of CPUs for control plane nodes in your management cluster, defaults to 2 if not set
-export VSPHERE_MEM_MIB="2048"                       # (optional) The memory (in MiB) for control plane nodes in your management cluster, defaults to 2048 if not set
-export SSH_AUTHORIZED_KEY="ssh-rsa AAAAB3N..."      # (optional) The public ssh authorized key on all machines in this cluster
+export VSPHERE_DATACENTER='SDDC-Datacenter'         # (required) The vSphere datacenter to deploy the management cluster on
+export VSPHERE_DATASTORE='DefaultDatastore'         # (required) The vSphere datastore to deploy the management cluster on
+export VSPHERE_NETWORK='vm-network-1'               # (required) The VM network to deploy the management cluster on
+export VSPHERE_RESOURCE_POOL='Resources'            # (required) The vSphere resource pool for your VMs
```

Comment:
Is this not optional?

Reply:
It is. I just copied stuff. I need to clean up this too, but it’s not necessarily part of this PR